player/Player.cs

```diff
@@ -6,17 +6,34 @@ public class Player : KinematicBody2D
     [Export] private float runningSpeed = 250f;
     [Export] private bool shouldSlide = false;
 
+    [Signal] public delegate void OnAction(Vector2 direction);
+
     private AnimationTree animationTree;
     private AnimationNodeStateMachinePlayback animationStateMachine;
     private bool isRunning = false;
+    private Vector2 facingDirection = Vector2.One;
 
     public override void _Ready()
     {
         base._Ready();
-        animationTree = (AnimationTree)GetNode("AnimationTree");
+        animationTree = GetNode<AnimationTree>("AnimationTree");
         animationStateMachine = (AnimationNodeStateMachinePlayback)animationTree.Get("parameters/playback");
     }
 
+    public override void _UnhandledInput(InputEvent @event)
```

Comment:
i reckon this should be `_UnhandledKeyInput` instead and (personal pref:) it should be moved to the bottom (after `_PhysicsProcess`)

Reply:
[75f76af](https://github.com/fourlastor-jams/game-off-2022/pull/13/commits/75f76afb2d7d068e55c43102cd6e89ebfd2d1f10) TIL!